railgun/test/integration/suite_test.go

```diff
@@ -155,11 +154,21 @@ func deployControllers(ctx context.Context, ready chan ktfkind.ProxyReadinessEve
 			}
 		} else {
 			config := manager.Config{}
-			flags := flag.NewFlagSet("", flag.ExitOnError)
-			manager.RegisterFlags(&config, flags)
+
+			flags := manager.MakeFlagSetFor(&config)
 			flags.Parse([]string{
 				fmt.Sprintf("--kong-url=http://%s:8001", adminHost),
 				fmt.Sprintf("--kubeconfig=%s", kubeconfig.Name()),
+				"--controller-kongstate=enabled",
+				"--controller-ingress-networkingv1=enabled",
+				"--controller-ingress-networkingv1beta1=disabled",
+				"--controller-ingress-extensionsv1beta1=disabled",
+				"--controller-udpingress=enabled",
+				"--controller-tcpingress=disabled",
```

Comment:
This API is enabled and tested currently in next, it would seem we need to enable it:
```suggestion
				"--controller-tcpingress=enabled",
```

Reply:
As discussed on zoom: the TCPingress should have failed with this setting but it didn't; the test will be fixed (and this flag will be updated) in a separate PR.